### ISSUE TITLE
Create Component / Banner

### ISSUE THREAD

**Issue by marcelbertram** [porsche-design-system/porsche-design-system#776]
Scope
As a user I want to receive prominent information on persistent changes / conditions that affects the entire system, i.e. disconnection / cookie consent. It is visible until actively dismissed or if the cause of the information is resolved.

**Questions**
- Do we need different importance hierarchy: error, success, warning, neutral? > Warning, Error + Neutral
- Do we need a close "X" or are buttons with label sufficient for dismissing?  > "X" by default and option to disable
- How is the page alignment with grid/content-wrapper: basic, extended, fluid? > Overlay
- Do we want to enable to include links for further information? > Text default but slotted content
- How do we handle showing multiple information "banners" in parallel? > Stacking by importance?

**Properties and variants**
- Text only
- Text and Close Icon

**Notes**  
- Fade in animation required for user attention
- Notificaton soft colors for dark
Error soft #FEC5C5 // RGB 254 / 197 / 197
Success soft #BFEEC6 // RGB 191 / 238 / 198
Warning soft #FFE6BF // RGB 255 / 230 / 191
Neutral soft #C7E4FF // RGB 199 / 228 / 255

**Out of scope**
- Cookie Banner and Marketing (inline) banner

**Link to resources**
- Docu: https://designsystem.porsche.com/issue/776/#/
- Overlay research: https://www.notion.so/porscheui/Research-Overlay-Components-1e1a68697ef64bafb89ba09dc119ecd5
- Notification Banner research: https://www.notion.so/porscheui/Notification-Banner-b7a03195ece248c7b5ec6ed70d283411
- Abstract Collection: https://share.goabstract.com/4929b647-bb69-45c8-be3d-a97689b8ae41
- Prototype: https://preview.uxpin.com/98682eed388d0db48e5c385724d12bee4756a639#/pages/132481740 (Desktop) + https://preview.uxpin.com/98682eed388d0db48e5c385724d12bee4756a639#/pages/132432315 (Mobil)

_Desktop_
Within Content 
Sticky Notification
Open Notification / From the top / Easy In Out Quadratic / Duration 600ms / Delay 1000s
Close Notification / Fade / Easy Out Quadratic / Duration 200ms / Delay 0ms

_Mobile_
Within Content 
Sticky Notification
Open Notification / From the bottom / Easy In Out Quadratic / Duration 600ms / Delay 1000s
Close Notification / Fade / Easy Out Quadratic / Duration 200ms / Delay 0ms

_Shadow_
1. [0, 2px 4px, RGBA(0,0,0,5)]
2. [0, 15px 20px, RGBA(0,0,0,20)]

Subtasks
---

**Design**
- [x] Collect use cases & do component research in notion
- [x] Create draft component in Sketch
- [x] Discuss Draft and Variants with the team
- [x] Research on real cases for text/content
- [x] Definition of naming, required states & variants
- [x] Define new color for cookie banner
- [x] Create new layer style in Basic Library
- [x] Create component in Sketch for light
- [x] Define notification soft color for dark
- [x] Write draft design docu
- [x] Create UX Pin prototype
- [x] Create component in Sketch for dark
- [x] Include notification soft color for dark theme
- [x] Refine design docu incl. images

**CODE**
- [x] tbd
